assets/images/Svg.js

```diff
@@ -38,7 +39,9 @@ const svg = {
   webpack: require("./webpack.svg")
 };
 
-module.exports = function(name, props) {
+const supportedLibraries = [{key: 'webpack', domain:'webpack://'}];
```

Comment:
Hmm, I think it is nice to keep `Svg` just about loading SVGs and move this logic to SourcesTree. There will be other cases where we want to show a webpack icon too